examples/tera_templates/templates/index.html.tera

```diff
@@ -2,12 +2,7 @@
 
 {% block content %}
     <h1>Hi {{name}}</h1>
-    <h3>Here are your items:</h3>
-    <ul>
-        {% for s in items %}
-            <li>{{ s }}</li>
-        {% endfor %}
-    </ul>
+    {% include "items" %}
```

Comment:
See my comment about using a `nav.html` as an include instead.

Reply:
Should I add an "about" page similar to the handlebars example, so the nav bar makes some sense? I don't really see a use for a nav bar with only one route.

Reply:
Sure! Let's have full parity, why not.

Reply:
Sorry I haven't realized there were other changes to the templating examples on master, an about page and a nav template already exists in the latest version, your comment makes much more sense know